tests/suites/test_suite_nist_kw.data

```diff
@@ -34,450 +33,458 @@ NIST KW lengths #3 KW ciphertext OK (3 to 2^54 semiblocks)
 nist_kw_ciphertext_lengths:32:24:MBEDTLS_KW_MODE_KW:0
 
 NIST KW lengths #4 KWP ciphertext OK (2 to 2^29 semiblocks)
-nist_kw_ciphertext_lengths:24:16:MBEDTLS_KW_MODE_KWP:0
+nist_kw_ciphertext_lengths:24:16:MBEDTLS_KW_MODE_KWP:PSA_ERROR_INVALID_PADDING
 
 NIST KW lengths #5 KW plaintext too short (2 to 2^54 - 1 semiblocks)
-nist_kw_plaintext_lengths:5:13:MBEDTLS_KW_MODE_KW:MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA
+nist_kw_plaintext_lengths:5:13:MBEDTLS_KW_MODE_KW:PSA_ERROR_DATA_INVALID
```

Comment:
For an output buffer that's too short, the PSA error is `PSA_ERROR_BUFFER_TOO_SMALL`.

Reply:
Sorry, I had missed your question here. [`PSA_ERROR_BUFFER_TOO_SMALL`](https://arm-software.github.io/psa-api/status-code/1.0/api/status-codes.html#c.PSA_ERROR_BUFFER_TOO_SMALL) specifically means that an output buffer is not large enough for the output data. The error `INVALID_ARGUMENT` conveys that some data is bad (whether it's an unknown value in an enum-like integer, a known value that isn't in an appropriate category, an integer out of range, a badly formatted string, etc.). If an error can be solved by allocating more memory for a buffer, without changing any data, that error should be conveyed by `BUFFER_TOO_SMALL`.